.github/workflows/windows_release.yml.unused

```diff
@@ -0,0 +1,136 @@
+name: Windows Releases
```

Comment:
Oh yeah should re-enable Windows.  And since they are all so closely related for this minimal build we could probably collapse them to a single workflow under 3 different jobs.